### PR TITLE
add didi slides

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: 20190618/sqlflow.slide [hunk 1]
```diff
@@ -0,0 +1,119 @@
+Contribute to SQLFlow
+18 June 2019
+
+Yan Xu
+Software Enginer, Ant Finatical
```
Comment (weiguoz) [inline 20190618/sqlflow.slide]: Software engineer, Ant Financial?

Comment (Yancey0623) [reply 20190618/sqlflow.slide]: thx fixed the typo.

**Comment by hungry1526** [comment]
My personal suggestions:
1.Use a more eye-catching title, something like, "SQLFlow project, Why and How to contribute"
2.Add a roadmap/nav page on the second page. 
3.Update the date, or remove. 
4.Add recognition page for our current team and the community, as a convention for open source development. 
5.Minor, use a slighter bigger title for each page. 
6. If possible, add some data/graphics to show the evolution of the community, stars, and contributions, so it is more visual to the audience.

File: 20190618/clone_sqlflow_models.bashsample [hunk 2]
```diff
@@ -0,0 +1 @@
+git clone git@github.com:<your Github account>/models.git
```
Comment (typhoonzero) [inline 20190618/clone_sqlflow_models.bashsample:1]: Is there any way that can use inline code in the `.slide` file? maybe indentation?

File: 20190618/custom_model.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+SELECT * FROM creditcardfraud
```
Comment (typhoonzero) [inline 20190618/custom_model.sql:1]: For figures like `column_all.png`, can just refer to the file using relative path: `../20190618/xx.png`

File: 20190618/sqlflow.slide [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+.code pred.sql
+
+* The Architecture
+
+.image sqlflow-overview.png 570 _
+
+* The Architecture
+
+- Users input SQL statements in Jupyter Notebook
+- The Jupyter plug-in sends each statement to SQLFlow server
```
Comment (typhoonzero) [inline 20190618/sqlflow.slide:38]: "Jupyter plug-in" => "SQLFlow magic command"

File: 20190618/sqlflow.slide [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+.code train_cross.sql
+
+And this will generate feature column code like:
+
+.code feature_column_cross.pysample
+
+* Open Source Community WorkFlow [Demonstration]
+.image github-workflow.png _ 860
+
+- Demonstration repo: https://github.com/Yancey1989/github_demo
```
Comment (typhoonzero) [inline 20190618/sqlflow.slide:101]: Better to not use personal repos.

Comment (Yancey0623) [reply 20190618/sqlflow.slide:101]: Moved to https://github.com/sql-machine-learning/sqlflow/pull/524 , thanks.

**Comment by Yancey0623** [comment]
Thanks @hungry1526 , I moved this PR to the sqlflow repo: https://github.com/sql-machine-learning/sqlflow/pull/524 . And update the first page to introduce SQLFlow community.

**Pull request closed without merging**